packages/components/src/Drawer/Drawer.component.js

```diff
@@ -102,37 +102,47 @@ export function cancelActionComponent(onCancelAction, getComponent) {
 	);
 }
 
-function renderSubtitleTag(subtitleTagLabel, subtitleTagTooltip) {
+function renderSubtitleTag(
+	subtitleTagLabel,
+	subtitleTagTooltip,
+	subtitleTagComponent = TagDefault,
+) {
 	if (subtitleTagTooltip) {
 		return (
 			<Tooltip placement="top" title={subtitleTagTooltip}>
-				<TagDefault>{subtitleTagLabel}</TagDefault>
+				{React.createElement(subtitleTagComponent, null, subtitleTagLabel)}
```

Comment:
```suggestion
				<Tag variant={subtitleTagVariant}>{subtitleTagLabel}</Tag>
```

Reply:
TagVariant is not exported by Tag module. Could I change it to add it ?

Reply:
Yes absolutely. You can even switch it from an enum to a union of strings. Enums un TS have little value.

Reply:
Updated, if you can review again please